internal/cli/atlas/backup/restores_start.go

```diff
@@ -151,6 +151,10 @@ func RestoresStartBuilder() *cobra.Command {
 		Short:     "Start a restore job for your project and cluster.",
 		Args:      require.ExactValidArgs(1),
 		ValidArgs: []string{automatedRestore, downloadRestore, pointInTimeRestore},
+		Annotations: map[string]string{
+			"args":     "deliveryType",
+			"nameDesc": "Type of restore job to create. Accepted values include: automated, download, pointInTime.",
+		},
```

Comment:
[q] is this correct? do we want to create an entry to the array for each value? @gssbzn

Reply:
I delegate this to @mongodb/docs-cloud-team but my preference for these cases where its a small set of known values we we don't annotate but make the long description explain them but up to docs as this is for their pages

Reply:
I'm still debating which way I think works better! For now, let's go with this. We will be fleshing out this content in Q3 and can revisit then.